python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_functional_chaos_exploitation.py

```diff
@@ -70,27 +139,30 @@
 
 # %%
 # Get the multivariate basis
-# as a collection of Function
+# as a collection of :class:`~openturns.Function`.
 reduced = result.getReducedBasis()
 
 # %%
-# Get the orthogonal basis
+# Get the orthogonal basis.
 orthgBasis = result.getOrthogonalBasis()
 
 # %%
-# Get the distribution of variables Z
+# Get the standard distribution :math:`\mu` of the standardized
+# random vector :math:`\vect{Z}`.
 orthgBasis.getMeasure()
```

Comment:
```suggestion
print(orthgBasis.getMeasure())
```

Reply:
I would rather not use `print` within an example, unless it is strictly necessary and here is why.

If the `print` statement is evaluated, then the C++ pretty-print of the object is used. This involves generating a string with ASCII-art table print:

![image](https://user-images.githubusercontent.com/31351465/236646695-686f67f2-5b4d-4396-8eb9-31b4012fbcb5.png)

If the implicit print is evaluated, then the Python pretty-print is used. This involves generating a HTML string with nice formatting:

![image](https://user-images.githubusercontent.com/31351465/236646718-fe26dc09-0515-4509-beac-d510cc616316.png)

Now, in #2282, I write a Python HTML pretty print of orthogonal basis. If that particular `print` is used, then the pretty-print will not be shown.

Reply:
@josephmure : You did not answer on this particular topic. Do you agree on this point?

Reply:
I do, thanks!